Source/Utils/WarpXUtil.H

```diff
@@ -128,4 +128,17 @@ T trilinear_interp(T x0, T x1,T y0, T y1, T z0, T z1,
 */
 WarpXParser makeParser (std::string const& parse_function, std::vector<std::string> const& varnames);
 
+namespace {
+// Map with default value (for not-present key)
```

Comment:
please promote this to a full-flavored doxygen block :) (see example link below)

please include `<map>` in this file.

Reply:
Documentation added.